control/patch.yaml

```diff
@@ -0,0 +1,41 @@
+openapi: 3.1.0
+
+info:
+  title: Control patch models
+  description: >-
+    All control patch schemas
+  version: ^0.0.0
+
+components:
+  schemas:
+    Control.Patch:
+      description: >-
+        Request for triggering action on resources in traffic generator
```

Comment:
description needs change?   this is not triggering action on resources?

Reply:
agreed, changed.